tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -295,22 +295,31 @@ def test__bump_and_persist_without_persist(self, test_repo):
 
     def test__update_timestamp(self, monkeypatch, test_repo):
         snapshot_version = 3
+        timestamp_version = 5
         fake_metafile = pretend.call_recorder(
             lambda *a, **kw: snapshot_version
         )
         monkeypatch.setattr(
             "repository_service_tuf_worker.repository.MetaFile", fake_metafile
         )
 
-        mocked_timestamp = pretend.stub(signed=pretend.stub(snapshot_meta=2))
+        mocked_timestamp = pretend.stub(
+            signed=pretend.stub(snapshot_meta=2, version=timestamp_version)
+        )
         test_repo._storage_backend.get = pretend.call_recorder(
             lambda *a: mocked_timestamp
         )
-        test_repo._bump_and_persist = pretend.call_recorder(lambda *a: None)
 
-        result = test_repo._update_timestamp(snapshot_version)
+        def fake__bump_and_persist(*a, **kw):
+            mocked_timestamp.signed.version += 1
```

Comment:
It's not a great idea or a good practice to access a variable outside the function when the same variable is given as an argument.
Better do it as you did in https://github.com/repository-service-tuf/repository-service-tuf-worker/pull/354 where you bump the metadata passed as an argument.